### PR TITLE
Increasing number of max open FD's through Systemd

### PR DESCRIPTION
**Pull request by Freeaqingme**
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/554)

### CODE REVIEW

File: recommended-production-settings.md [hunk 1]
```diff
@@ -242,4 +258,4 @@ CockroachDB does not yet provide a native Windows binary. Once that's available,
 
 #### Attributions
 
-This section, "File Descriptors Limit", is a derivative of [Open File Limits](http://docs.basho.com/riak/kv/2.1.4/using/performance/open-files-limit/) by Riak, used under Creative Commons Attribution 3.0 Unported License.
\ No newline at end of file
+This section, "File Descriptors Limit", is a derivative of [Open File Limits](http://docs.basho.com/riak/kv/2.1.4/using/performance/open-files-limit/) by Riak, used under Creative Commons Attribution 3.0 Unported License.
```
Comment (Freeaqingme) [inline recommended-production-settings.md]: \ No newline at end of file

**Comment by jseldess** [comment]
Thanks for this contribution, @Freeaqingme! @bdarnell or @BramGruneir, would you please give this a quick review?

**Comment by bdarnell** [comment]
LGTM, although I suspect that organizationally we'll eventually want to have a page or section about running cockroachdb under systemd with a complete service definition, instead of a systemd section nested under changing file descriptor limits. (for example, the cockroachdb service should be `type=notify`)

**Comment by Freeaqingme** [comment]
Ideally I guess it'd probably be easier to simply add a 'dist' dir to the cockroach repo that contains things like Debian and Rhel packaging scripts, as well as one systemd service definition that contains all best practices.

Then at relevant spots you could just note "If you're using our packages or the systemd service definition found <here> this is all done for you" or something along those lines.

But, given that such a thing probably becomes more relevant once you're reaching GA, I figured this would make due. 

Also, as noted on Gitter, there is one failed CircleCI item of which I'm unsure what is actually failing and if this is something I can fix. Any guidance on this would be appreciated.

**Comment by jseldess** [comment]
LGTM, @Freeaqingme. I'll merge now and do a little copy tweaking in the morning before pushing it live.

**Pull request merged**